crates/swc_ecma_minifier/tests/benches-full/d3.js

```diff
@@ -1969,19 +1969,19 @@
         var t0, i0;
         function tween() {
             var fullname1, i, i1 = value.apply(this, arguments);
```

Comment:
It looks like `fullname1` is unused?

Reply:
Typically we did it in another PR because dropping variables require more analysis, but in this case, I think we can do it within this PR.

We can leverage `Finalizer` or drop variables directly when we inline I think.

Reply:
For lazily inited vars, we need some extra works to remove it in one pass, because at the time of inline we don't have the mutable reference to var declaration. I believe the most optimal way is to run DCE pass at the end but that seems to be bad for performance somehow.

Reply:
For lazily inited vars, we need some extra works to remove it in one pass, because at the time of inline we don't have the mutable reference to var declaration. I believe the most optimal way is to run DCE pass at the end but that seems to be bad for performance somehow.

Reply:
Can't we add it to the removal list of `Finalizer`?

https://github.com/swc-project/swc/blob/4f92225db78ed796fdba75328821a4914997de9e/crates/swc_ecma_minifier/src/compress/optimize/util.rs#L168

Sequential pass removes variables in a single pass by using `Finalizer`.

Reply:
Fine. But I'd prefer to do it in a seperate PR as it involves around 800 lines of change.

Reply:
Then can you rebase?